database/schema/create_schema.sql

```diff
@@ -620,7 +618,8 @@ CREATE TABLE IF NOT EXISTS advisory_account_data
     CONSTRAINT status_id
         FOREIGN KEY (status_id)
             REFERENCES status (id),
-    UNIQUE (advisory_id, rh_account_id)
+    UNIQUE (advisory_id, rh_account_id),
```

Comment:
Why are we leaving UNIQUE constraint here? Couldn't we remove it like in system_advisories table because of primary key couple?

Reply:
The unique constraint creates `advisory_id, rh-account_id` index, and the primary key creates
`rh_account_id, advisory_id` index.